components/docs-chef-io/content/automate/troubleshooting.md

```diff
@@ -180,3 +180,39 @@ max_shards_per_node = 1000
 ```
 
 Once done, run the chef-automate config patch `</path/to/your-file.toml>` to deploy your change.
+
+## Issue: Knife Search Limits at 10000 Records
+
+### Details
+
+The knife node list and knife node search commands are inconsistent in terms of the number of records they return. Specifically, knife search is limited to a maximum of **10000** records by default.
+
+### Fixes
+
+This happens because OpenSearch,by default, limits the maximum number of records(or document) returned in a single query to **10000**. This is a safeguard to prevent large queries from overloading the system. If you are trying to retrive more than **10000** records, this approach will do that.
```

Comment:
missing a space after , and before by
missing a space after records and before (
document should be documents, surely?
retrive should be retrieve

Reply:
Thank you for reviewing. I pushed the fix.